conda-forge.yml

```
@@ -1,9 +1,14 @@
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

These can stay `default`

```suggestion
provider:
  linux_aarch64: default
  linux_ppc64le: default
```